govcd/vm.go

```diff
@@ -727,6 +727,18 @@ func (vm *VM) GetQuestion() (types.VmPendingQuestion, error) {
 
 }
 
+func (vm *VM) GetMetadata(requestUri string) (*types.Metadata, error) {
+	return getMetadata(vm.client, requestUri)
+}
+
+func (vm *VM) DeleteMetadata(key string) (Task, error) {
+	return deleteMetadata(vm.client, key, vm.VM.HREF)
+}
+
+func (vm *VM) AddMetadata(key string, value string) (Task, error) {
```

Comment:
Please add an SDK-user targeted comment (documentation) about the three public functions above.

Reply:
Fixed.